src/vosk_api.cc

```diff
@@ -100,3 +100,22 @@ void vosk_set_log_level(int log_level)
 {
     SetVerboseLevel(log_level);
 }
+
+#if HAVE_CUDA
+#include "cudamatrix/cu-device.h"
+#endif
+
+void vosk_gpu_init()
+{
+    #if HAVE_CUDA
```

Comment:
preprocessor directives usually start from beginning of the line without indent.

Reply:
Will fix.

Reply:
Reformatted.